scripts/lib.sh

```diff
@@ -85,3 +88,68 @@ develop() {
     install-js-dev
     install-py-dev
 }
+
+init-config() {
+    sentry init --dev
+}
+
+run-dependent-services() {
+    sentry devservices up
+}
+
+create-db() {
```

Comment:
Porting `create-db` and `drop-db` are the only part of this PR that required few tries to port it.